docs/localization/LOCALIZATION.md

```diff
@@ -419,107 +523,172 @@ class AppModule {}
 
 > [!NOTE]
 > The token `MESSAGE_FORMAT_CONFIG` implements the `LazyMessageFormatConfig` interface from `@o3r/localization`.
-git rebase --continue
 
-> [!TIP]
-> The full documentation about MessageFormat configuration is available on <https://github.com/lephyrus/ngx-translate-messageformat-compiler>.
+### How to localize plural expressions
 
-### How to localize plural expression
+For pluralization, we are using the `TranslateMessageFormatCompiler`, coming from the [ngx-translate-messageformat-compiler](https://www.npmjs.com/package/ngx-translate-messageformat-compiler) package, which is a compiler for `ngx-translate` that uses `messageformat.js` to compile translations using ICU syntax for handling pluralization and gender.
 
-For pluralizations, we are using *TranslateMessageFormatCompiler*, coming from [ngx-translate-messageformat-compiler](https://www.npmjs.com/package/ngx-translate-messageformat-compiler) package, which is a compiler for *ngx-translate* that uses *messageformat.js* to compile translations using ICU syntax for handling pluralization and gender.
 [ICU Message Format](http://userguide.icu-project.org/formatparse/messages) is a standardized syntax for dealing with the translation of user-visible strings into various languages that may have different requirements for the correct declension of words (e.g. according to number, gender, case) - or to simplify: pluralization.
```

Comment:
the link is not accessible anymore
maybe could be replaced by https://messageformat.github.io/messageformat/guide/